index.html

```diff
@@ -0,0 +1,139 @@
+
+      <article class="intro-align">
+    <div class="crypto-text" id="intro">"Blockchain crypto!"</div>
+
+    
+    <h1 class="ico-title">NEW CRYPTO COIN 2022 INITIAL COIN OFFERING</h1>
+    <div class="ico-words"> The crypto currency that has come to create financial freedom and be the foundation of the future of finance,all over the world. The ico will be taking place on December .</div>
+    <h2 class="ico-date">2022.12.01(TUE)~30(FRI)</h2>
+    <div id="contact" class="contact">@ www.fakenewcryptocoin.com</div>
+      </article>
+
+    </section>
+
+
+    <section class="main-section">
+        
+    <h2 class="program-class">Main Program</h2>
+        
+        <div class="card-flex">
+            
+      <div class="card">
+        <img src="./images/download.jpeg" alt="whitepaper image" class="program-img">
+        <div class="card-container">
+          <h4><b>Release of official complete white paper</b></h4>
+        </div>
+
+      </div>
+
+      <div class="card">
+        <img src="./images/download.png" alt="explanation" class="program-img">
+        <div class="card-container">
+          <h4><b>Explanation of whitepaper</b></h4>
+        </div>
+      </div>
+      
+        <div class="card">
+            <img src="./images/download (1).jpeg" alt="roadmap" class="program-img">
+            <div class="card-container">
+              <h4><b>official roadmap</b></h4>
+            </div>
+        
+        </div>
+      </div>
+        </div>
+        
+        <form action="https://formspree.io/f/xbjbawkn" method="post" id="form">
+            <input type="text" name="Name" maxlength="30" placeholder="Name">
+    
+            <div class="submit">
+            Please type/text your email here to receive authentication email to participate in the ICO.
+            <textarea name="e-mail" cols="40" rows="1" placeholder="E-mail"></textarea>
+            <button type="submit" value="Submit">Submit</button>
+          </div>
+        </form> 
+    </section>
+
+    
+
+    <section class="speaker-section">
+        
+        <ul class="speakers-list">
+    <h2>Featured Speakers</h2>
+    <hr class="speaker-line1">
+
+    <div class="list-container">
+    
+       
+    </div>
+
+    </div>
+    </ul>
+    
+    <div class="more-button"><button class="more-width">MORE</button></div>
+    
+    </section> 
+
+    <section>
+        <div class="partners-class">
+            <h3 class="partners">Partners</h3>
+            <hr class="speaker-line1" />
+            <ul class="sponsors-list">
+              <li>Mozilla</li>
+              <li>Google</li>
+              <li>NAVER</li>
+              <li>daumkakao</li>
+              <li>airbnb</li>
+            </ul>
+          </div>
+
+        <footer class="footer1">
+        <img src="./images/240957a3-789c-4e2b-9fc7-34a08b3e9f0c.png" class="footer-image"alt="">
+        <div>2020 crypto commons reserved.Some rights reserved.</div>
+        </footer>
+
```

Comment:
On Desktop design: Footer is tooo large. Kindly try to get closer to the design provided.
| Your Design   | FIGMA Design  |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/61361037/219805782-1ada4c4b-38d8-4530-96ff-a5ee58e4bb5f.png) | ![image](https://user-images.githubusercontent.com/61361037/219806294-8d148f7b-909b-478c-aa1e-1403612ae32a.png)|